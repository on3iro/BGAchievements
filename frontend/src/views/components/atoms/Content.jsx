// @flow

import styled from 'styled-components'

const Content = styled.div`
  flex-grow: 1;

  padding: 56px 24px 24px;
  min-height: calc(100% - 56px);
`

export {
  Content as default
}
