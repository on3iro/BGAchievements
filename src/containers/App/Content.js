/**
  * Styled Wrapper for site content
  *
  * @namespace App.Content
  * @memberOf App
  */

import styled from 'styled-components'

const Content = styled.main`
  margin: 10px auto 60px auto;
  padding: 0;
  width: 100%;
  height: 60%;

  @media (max-width: 900px) {
    padding: 20px;
    height: 100%;
  }
`

export default Content
