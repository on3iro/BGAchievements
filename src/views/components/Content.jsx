// @flow

import styled from 'styled-components'

const Content = styled.div`
  flex-grow: 1;

  padding: 20px;
  max-width: 60%;
  min-height: calc(100% - 40px);
`

export {
  Content as default
}
