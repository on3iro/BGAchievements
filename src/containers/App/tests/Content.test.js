import React from 'react'
import { shallow } from 'enzyme'

import Content from '../Content'

describe('<Content />', () => {
  it('Should render main', () => {
    const wrapper = shallow(<Content />)
    expect(wrapper.find('main').length).toMatchSnapshot()
  })
})
