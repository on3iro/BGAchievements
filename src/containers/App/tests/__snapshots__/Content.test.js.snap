exports[`<Content /> Should render main 1`] = `1`
